CONTRIBUTING.rst

```diff
@@ -187,7 +188,7 @@ These are the basic steps needed to start developing on Sphinx.
        git push origin feature-xyz
 
 #. Submit a pull request from your branch to the respective branch (``master``
-   or ``stable``) on ``sphinx-doc/sphinx`` using the GitHub interface.
+   or ``X.Y``) on ``sphinx-doc/sphinx`` using the GitHub interface.
```

Comment:
You can drop everything after the `)` here: if user's don't know how to submit a pull request, I doubt they'll be contributing.